bundle/metadata/annotations.yaml

```diff
@@ -6,7 +6,7 @@ annotations:
   operators.operatorframework.io.bundle.package.v1: node-healthcheck-operator
   operators.operatorframework.io.bundle.channels.v1: candidate
   operators.operatorframework.io.bundle.channel.default.v1: candidate
-  operators.operatorframework.io.metrics.builder: operator-sdk-v1.14.0+git
+  operators.operatorframework.io.metrics.builder: operator-sdk-v1.18.0+git
```

Comment:
1.17 as well, see my comment above.

Reply:
It is fine, I haven't noticed there is a new version of Operator-sdk (see [1.19](https://github.com/operator-framework/operator-sdk/releases/tag/v1.19.0)).